pkg/social/social.go

```diff
@@ -91,10 +97,15 @@ func NewOAuthService() {
 			Scopes:      info.Scopes,
 		}
 
+		logger := log.New("oauth.login." + name)
```

Comment:
The logging framework support hierarchy.

ex 
```golang
logger := log.New("oauth.login")
oauthlogger = logger.New("name", "github")

oauthlogger.Info("Great success!") 
```
> "message"="Greate success!", "logger"="oauth.login", "name"="github"

Reply:
that looks strange though, the logger string should be "oauth.login.github" in that case, I think just "oauth." + name is good as well. "login" feels a bit unnecessary (I know the oauthLogger in the api uses "oauth.login" though but that seems to be only used (currently) on one place so that could be renamed to "oauth" maybe or "login.auth"

Reply:
I think it makes sense to include the OAuth provider name since it can make it easier to debug a setup with multiple OAuth providers. Otherwise, we will end up adding the name of the provider in all log statements.